internal/datanode/flow_graph_insert_buffer_node.go

```diff
@@ -502,6 +506,7 @@ func (ibNode *insertBufferNode) Sync(fgMsg *flowGraphMsg, seg2Upload []UniqueID,
 		ibNode.channel.rollInsertBuffer(task.segmentID)
 		ibNode.channel.RollPKstats(task.segmentID, pkStats)
 		ibNode.channel.setSegmentLastSyncTs(task.segmentID, endPosition.GetTimestamp())
+		ibNode.channel.(*ChannelMeta).needToSync.Store(false)
```

Comment:
better to move to head of this function ?

Reply:
done